src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/Endpoint.kt

```diff
@@ -17,7 +18,7 @@ data class Endpoint(
         override val actionName: String,
         override val contentType: String = ContentTypes.json,
         override val method: HttpMethod = HttpMethod.get,
-        override val transform: Boolean = false,
```

Comment:
The `transform` property wasn't used anywhere, so I removed it while adding the new one, to keep complexity down.